src/suricata.h

```diff
@@ -214,6 +216,11 @@ SCRunMode SCRunmodeGet(void);
  */
 void SCRunmodeSet(SCRunMode run_mode);
 
+/**
+ * \brief Enable default signal handlers.
+ */
+void SCEnableDefaultSignalHandlers(void);
```

Comment:
In 9.0, we'll probably expand on the SCInstance, or some SCInstanceConfig (SCEngineConfig) that will be passed as a parameter here, as part of the work to allow multiple instances, should we desire that (but just seems like a good idea for a library).